gallery/src/components/NxToast/NxToastWithNxDrawerExample.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState, ComponentType } from 'react';
+
+import { reject, propEq } from 'ramda';
+
+import {
+  NxDrawer,
+  NxButton,
+  useToggle,
+  NxBackButton,
+  NxPageTitle,
+  NxH1,
+  NxPageMain,
+  NxP,
+  NxToastContainer,
+  NxToast,
+  NxErrorAlert,
+  NxSuccessAlert
+} from '@sonatype/react-shared-components';
+
+interface ToastModel {
+  id: number;
+  alertComponent: ComponentType;
+  message: string;
+}
+
+export default function NxDrawerWithNxToastExample() {
```

Comment:
Is there any meaningful difference between the "toast with drawer" example and the "drawer with toast" example? I don't think we need both.  Personally I would have only put the example on one of the two documentation pages, but even if you have it on both, they could both link to the same actual example page.

Reply:
The only difference is so the 'Back to Documentation' button links back to the drawer or toast page depending on where it was accessed from. I initially had it as just one example, but the Back to Documentation link from the toast page would go back to the Drawer page instead, but if that isn't a problem I can just link them to the same example.

Reply:
> The only difference is so the 'Back to Documentation' button links back to the drawer or toast page depending on where it was accessed from. I initially had it as just one example, but the Back to Documentation link from the toast page would go back to the Drawer page instead, but if that isn't a problem I can just link them to the same example.

Oh that's a good point.  Just leave it like this then.  There are ways to address that that could result in less code duplication, but at the expense of making the example code more complicated which is undesirable since they are examples